warp-contracts-plugin-quickjs/src/eval/QuickJsEvaluator.ts

```diff
@@ -78,12 +85,26 @@ export class QuickJsEvaluator {
     const processId = this.vm.getString(processIdHandle);
     const action = this.vm.getString(actionHandle);
     const { dryrun } = await import('@permaweb/aoconnect');
-    const readRes = await dryrun({
-      process: processId,
-      tags: [{ name: 'Action', value: action }],
-      data: '1234'
-    });
-    return JSON.parse(readRes.Messages[0].Data);
+    try {
+      const result = await Promise.race<{
```

Comment:
the issue with race is that the 'timeout' promise won't be remove even if the 'raced' promise is settled.
https://stackoverflow.com/a/39852372

It is already handled by a timeout implementation in warp sdk - https://github.com/warp-contracts/warp/blob/main/src/utils/utils.ts#L52